cli/pcluster/cli.py

```diff
@@ -99,8 +100,9 @@ def config_logger():
 
     log_file_handler = RotatingFileHandler(logfile, maxBytes=5 * 1024 * 1024, backupCount=1)
     log_file_handler.setLevel(logging.DEBUG)
-    log_file_handler.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(name)s - %(message)s"))
+    log_file_handler.setFormatter(logging.Formatter("%(asctime)s - %(levelname)s - %(module)s - %(message)s"))
```

Comment:
Just out of curiosity, what is the goal of this change?

Reply:
It's to always have the name of the module that originated the log record rather than the name of the logger that can be arbitrary